efi/internal/default_env_test.go

```diff
@@ -39,7 +40,11 @@ type defaultEnvSuite struct{}
 var _ = Suite(&defaultEnvSuite{})
 
 func (s *defaultEnvSuite) TestVarContext(c *C) {
-	c.Check(DefaultEnv.VarContext(), Equals, efi.DefaultVarContext)
+	ctx := DefaultEnv.VarContext(context.Background())
+	c.Assert(ctx, NotNil)
+
+	expected := efi.WithDefaultVarsBackend(context.Background())
```

Comment:
maybe a good idea to pass something with an attached value already and check it can be retrieved?

Reply:
I've added this to the test